app/scripts/lib/snap-keyring/snap-keyring.test.ts

```diff
@@ -68,7 +67,7 @@ const createControllerMessenger = ({
     SnapKeyringBuilderAllowActions,
     never
   >().getRestricted({
-    name: 'SnapKeyringBuilder',
+    name: 'SnapKeyring',
```

Comment:
We have to update this since the `AccountsController` uses this messenger's name now for the new keyring events.